src/data/finset/basic.lean

```diff
@@ -2454,8 +2458,9 @@ protected def finset_congr (e : α ≃ β) : finset α ≃ finset β :=
 @[simp] lemma finset_congr_apply (e : α ≃ β) (s : finset α) :
   e.finset_congr s = s.map e.to_embedding :=
 rfl
-@[simp] lemma finset_congr_symm_apply (e : α ≃ β) (s : finset β) :
-  e.finset_congr.symm s = s.map e.symm.to_embedding :=
+
+@[simp] lemma finset_congr_symm (e : α ≃ β) :
+  e.finset_congr.symm = e.symm.finset_congr :=
 rfl
```

Comment:
While you're here, could you also add the `trans` and `refl` lemmas,
```suggestion
@[simp] lemma finset_congr_refl :
  (equiv.refl α).finset_congr = equiv.refl _ :=
rfl

@[simp] lemma finset_congr_symm (e : α ≃ β) :
  e.finset_congr.symm = e.symm.finset_congr :=
rfl

@[simp] lemma finset_congr_trans (e : α ≃ β) (e' :  β ≃ \gamma) :
  e.finset_congr.trans (e'.finset_congr) = (e.trans e').finset_congr :=
rfl
```
(untested, but should be close)